src/app/App.svelte

```diff
@@ -7,18 +7,32 @@
 
 <style>
   :global(:root) {
-    --blue: #6fbbd4;
-    --blue-light: #b3dbe7;
-    --red: #b90529;
-    --red-light: #de929c;
-    --orange: #f18407;
-    --orange-light: #fbc88f;
+    /* Keyman brand colours, derived from https://keyman.com/ */
+    --blue: #6FBBD4;
+    --blue-light: #B3DBE7;
+    --red: #B90529;
+    --red-light: #DE929C;
+    --orange: #F18407;
+    --orange-light: #FBC88F;
     --gray: #888888;
-    --gray-light: #d6d6d6;
+    --gray-light: #D6D6D6;
+
+    /* Font stack */
+    /* Custom fonts downloaded from Google Fonts -- see public/index.html */
+    --main-font: "Cabin";
+    --mono-font: "Roboto Mono";
   }
 
-  :global(:root) {
-    font-family: "Carbin", sans-serif;
+  :global(body) {
+    font-family: var(--main-font), sans-serif;
+  }
+
+  :global(h1, h2, h3, h4, h5, h6) {
+    font-family: var(--main-font), sans-serif;
+  }
+
+  :global(pre, code, samp) {
+    font-family: var(--mono-font), monospace;
   }
```

Comment:
**Question**: Do we have global stying for body text? I would strongly recommend implementing a utility class for the purpose of having a clean and tidy looking css. Other than that we could also package our global styling to make this global value file looks cleaner

Reply:
Yeah, I think I'd prefer a global CSS file rather than tucking the CSS within each component, but I'll create a new issue for that!

If we want to incorporate that with Svelte component styles, we'll need a bit more work...